### DISCUSSION TITLE
Can we have a release of jbang-maven-plugin?

### DISCUSSION THREAD

**Discussion by cstamas** [jbangdev/jbang#1720] [General]
Would like to ask to have a release of jbang-maven-plugin. Thanks!

@gastaldi

**Comment by gastaldi** [comment]
That's up to @maxandersen to do, I don't have enough permission to perform it 😞

**Reply by maxandersen** [reply]
Been two years...have to remember :)

**Comment by maxandersen** [comment]
so seems like I did setup it up for automatic publish...but things changed so new requirements:

https://github.com/jbangdev/jbang-maven-plugin/actions/runs/7307793516/job/19913828143#step:4:4490

add info to readme.md about release but need to figure what need to change to get those new sign requirement done.

**Reply by cstamas** [reply]
Profile: gpg-sign. Signing was required since eons on central

**Reply by maxandersen** [reply]
Don't know what to tell ya. Didn't seem required 2 years ago :)

**Reply by gastaldi** [reply]
@maxandersen are you sure the `release` profile is enabled when the release is performed?

**Reply by maxandersen** [reply]
Yeah otherwise sonatype wouldn't be complaining about closing of the release.

**Reply by gastaldi** [reply]
Maybe the `gpg-sign` profile then, this is defined in jboss-parent: https://github.com/jboss/jboss-parent-pom/blob/main/pom.xml#L860C1-L860C1

**Reply by maxandersen** [reply]
you were right @gastaldi - -Drelease was not enabled which would have attempted sign ....but would still fail :/

so for now i've done it locally:

`MAVEN_USERNAME=jbangbot MAVEN_PASSWORD=SECRET mvn -Drelease -Dgpg.keyname="jbangbot@jbang.dev" -Dgpg.passphrase=OTHERSECRET clean deploy -DskipTests`

should make this simpler...also did not realize we got dependency to jboss parent. I would prefer that get cut out eventually. Don't think we (should) need it.
